README.rst

```diff
@@ -77,6 +77,20 @@ Then to upload content create a CSV or JSON file ::
 
 Find demo_import_identities.csv and .json in the repo route for format example.
 
+Identities Details Update
+-------------------------
```

Comment:
I think we should make it clear in the docs here that this is for updating the values in identity details, not the keys.